chunk_downloader.go

```diff
@@ -65,7 +65,7 @@ type snowflakeChunkDownloader struct {
 	RowSet             rowSetType
 	FuncDownload       func(context.Context, *snowflakeChunkDownloader, int)
 	FuncDownloadHelper func(context.Context, *snowflakeChunkDownloader, int) error
-	FuncGet            func(context.Context, *snowflakeChunkDownloader, string, map[string]string, time.Duration) (*http.Response, error)
+	FuncGet            func(context.Context, *snowflakeConn, string, map[string]string, time.Duration) (*http.Response, error)
```

Comment:
Can we add another method? I believe once interface publish not a good idea to change the method signature.

Reply:
I'm not sure what you mean, this interface isn't a public one, so it shouldn't be an issue to change the signature, right?